app/models/chargeback.rb

```diff
@@ -13,6 +13,10 @@ class Chargeback < ActsAsArModel
     :metering_used_cost   => :float
   )
 
+  def self.self_class
+    Settings[:new_chargeback] ? include(Chargeback::New) : self
```

Comment:
With settings, just use dot notation... `Settings.new_chargeback`  Additionally, this setting should be under the prototype section...  Settings.protoype.new_chargeback